refactor:

ntretyakov1 opened the pull request:
1.Нейминг констант;
2.функция "calculateCost" больше не использует api;
3.Вынес в константы значения день в секундах и миллисекунды в секундах;

ntretyakov1 committed:
refactor:
1.Нейминг констант;
2.функция "calculateCost" больше не испульзует api;
3.Вынес в константы значения день в секундах и миллисекунды в секундах;

KolesnichenkoAleksey commented on const.js:
```diff
@@ -1,27 +1,41 @@
+	EnumLaserRemovalOfBloodVessels: 233761,
+	EnumMimicWrinklesCorrection: 233763,
+	EnumLaserHairRemoval: 233765,
+};
```
Я бы сделал немного по-другому:
const DealCustomFieldServices = {
    FieldId: 432917,
	ServicesIds: {
        LaserFacialRejuvenation: 233757,
	    UltrasonicLifting: 233759,
	    LaserRemovalOfBloodVessels: 233761,
	    MimicWrinklesCorrection: 233763,
	    LaserHairRemoval: 233765,
    }
};

KolesnichenkoAleksey commented on const.js:
```diff
@@ -1,27 +1,41 @@
+	EnumMimicWrinklesCorrection: 233763,
+	EnumLaserHairRemoval: 233765,
+};
+const ContactIdFieldsServices = {
+	FieldLaserFacialRejuvenation: 432753,
+	FieldUltrasonicLifting: 457003,
+	FieldLaserRemovalOfBloodVessels: 457005,
+	FieldMimicWrinklesCorrection: 456951,
+	FieldLaserHairRemoval: 456953,
+};
+
```
Зачем каждому полю приписка "field", если из названия объекта можно понять, что это "Fields", единственное что я бы поменял название объекта на ContactFieldsServicesIds, чтобы явно показать что перечисление хранит в себе именно ID'шники

KolesnichenkoAleksey commented on index.js:
```diff
@@ -25,10 +30,14 @@ app.post("/lead", async (req, res) => {
 
 		if (!contacts.length) {
-			return res.send("OK");
+			return res.send("No contacts");
```
Можно не править, но просто, чтобы ты знал, что так можно res.status(404).send("Contacts not found!"), типа выкидываешь код ошибки, чтобы с клиентской стороны было проще понять, где именно ошибся сервер

ntretyakov1 committed:
fix:
1.Исправил значение в константе "MILLISECONDS_PER_SECONDS";
refactor:
1.Изменил нейминг констант на более понятный.
